docs/managing-e2e-tests.md

```diff
@@ -23,106 +24,140 @@ It contains a list of jobs such as:
+It contains a folder with workflows that need update each cycle:
+https://github.com/kubernetes/kubeadm/blob/master/kinder/ci/workflows/
+
+The summary of the actions is the following:
+- If there are `1.12 -> 1.13` upgrade jobs leave them, because upgrades to `1.13` still have to be supported.
+- Remove `1.13 on 1.12` jobs.
+- Remove plain `1.12` jobs.
+- Add upgrade jobs for `1.14 -> 1.15` and `1.15 on 1.14` skew jobs.
+- Add plain `1.15` jobs.
+- Make sure that workflows include the correct `ci/latest-x` labels.
+- Make sure that workflows have the correct testgrid URLs.
 
-This document will cover information on how to perform updates on these 3 files:
-- [kubeadm-kind.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml "kubeadm-kind.yaml")
+#### Updating test jobs in kubernetes/test-infra
+
+This document will cover information on how to perform updates on these files:
+- [kubeadm-kind.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml)
 
 Holds test jobs where the kubeadm version matches the Kubernetes control-plane and the kubelet versions.
 
-- [kubeadm-kinder-upgrade.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade.yaml "kubeadm-kinder-upgrade.yaml")
+- [kubeadm-kinder-upgrade.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade.yaml)
 
 Holds test jobs that perform a upgrade from version X to version Y (usually `Y = X + 1`).
 
-- [kubeadm-kinder-x-on-y.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-x-on-y.yaml "kubeadm-kinder-x-on-y.yaml")
+- [kubeadm-kinder-x-on-y.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-x-on-y.yaml)
 
 Holds test jobs that run kubeadm version X, on a control plane and kubelet version Y,
 as kubeadm does support `Y = X - 1`
 
+- [kubeadm-kinder-external-etcd.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-external-etcd.yaml)
+
+Holds tests that deploy a kubeadm / kinder cluster using external etcd, instead of the default stacked etcd.
+
 The files can be found in the [sig-cluster-lifecycle](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/sig-cluster-lifecycle) folder.
 
-The idea is to remove old test jobs and add new ones. If Kubernetes 1.15 is about to be released soon, this means that the `15 - 3 = 12` MINOR version would go outside of the support skew of the project. `1.12` jobs have to be removed and `1.15` jobs have to be added. The project would support version `15 - 2 = 13` as the minimum.
 
+The summary of the actions is the following:
```

Comment:
This list, at least at the begging, is the same as the one above under "Updating kinder workflows".
What about leaving only recommendations for creating new jobs?

Reply:
consolidated the list of actions:
```
### Updates to kubeadm tests
...
The summary of the actions is the following:
- shared actions here

#### Updating kinder workflows
- specific actions to kinder

#### Updating test jobs in kubernetes/test-infra
- specific actions to test infra

```